component/admin/models/media.php

```diff
@@ -285,9 +294,12 @@ public function deleteAddtionalFiles($fileId)
     public function saveorder($cid = array(), $order)
     {
         $row        = $this->getTable('media_detail');
-        $order      = JFactory::getApplication()->input->post->get('order', array(0), 'array');
         $conditions = array();
-
+        if (!isset($order) || $order == array())
+        {
+            $order  = JFactory::getApplication()->input->post->get('order', array(0), 'array');
```

Comment:
@hoangvan149dhv It's better for using namespace `\Joomla\...\Factory` then class with prefix `J` for best pratice that compatible with next version of Joomla.